src/Classes/DBPopulator.php

```diff
@@ -0,0 +1,81 @@
+<?php
+namespace ListingsApp\Classes;
+class DBPopulator {
+    private object $listingAPI;
+    private object $dbConnection;
+    private array $listings;
+    public function __construct($listingAPI, $db) {
```

Comment:
Missing doc block and line of whitespace above the construct. 

Also please type hint both params